src/evaluators/CallExpression.js

```diff
@@ -302,15 +300,39 @@ function EvaluateCall(
 ): Value {
   if (func instanceof AbstractValue) {
     let loc = ast.callee.type === "MemberExpression" ? ast.callee.property.loc : ast.callee.loc;
+    if (func.kind === "conditional") {
+      let [condValue, consequentVal, alternateVal] = func.args;
+      invariant(condValue instanceof AbstractValue);
+      if (
+        Value.isTypeCompatibleWith(consequentVal.getType(), FunctionValue) ||
```

Comment:
Why || and not &&?

Reply:
It's an optimization, if neither values are functions then just proceed with the default code path.